crates/rbuilder/src/building/payout_tx.rs

```diff
@@ -53,22 +53,23 @@ pub fn insert_test_payout_tx(
 
     let nonce = state.nonce(builder_signer.address)?;
 
-    let mut cfg = ctx.evm_env.cfg_env.clone();
-    // disable balance check so we can estimate the gas cost without having any funds
-    cfg.disable_balance_check = true;
-
+    let tx_value = 10u128.pow(18); // 10 ether
     let tx = create_payout_tx(
         ctx.chain_spec.as_ref(),
         ctx.evm_env.block_env.basefee,
         builder_signer,
         nonce,
         to,
         gas_limit,
-        0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF,
+        U256::from(tx_value),
     )?;
 
     let mut db = state.new_db_ref();
     let mut evm = ctx.evm_factory.create_evm(db.as_mut(), ctx.evm_env.clone());
+
+    let cache_account = evm.db_mut().load_cache_account(builder_signer.address)?;
+    cache_account.increment_balance(tx_value * 2); // double to cover tx value and fee
```

Comment:
What would happen if account balance is 0 and tx value is really slow?
We simulate top of block with balance is 0 (random coinbase) and this can break some refund bundles.

I think its safer to have value + some big value like base fee * 30mil